src/spec.c

```diff
@@ -107,13 +114,18 @@ crun_command_spec (struct crun_global_arguments *global_args, int argc, char **a
         libcrun_fail_with_error (errno, "chdir `%s` failed", bundle);
     }
 
-  ret = access ("config.json", F_OK);
-  if (ret == 0)
-    return libcrun_make_error (err, 0, "config.json already exists", err);
+  where = fname ? fname : "config.json";
+
+  if (fname == NULL)
```

Comment:
nit, not-a-blocker: I think we dont need `fname== NULL` since `where` already contains either a valid file or `config.json`

Reply:
sure, but I want to enable the check only when there was no `--file` specified to behave similarly to runc

Reply:
oh i see.